src/pkg/cli/compose.go

```diff
@@ -107,15 +107,64 @@ func warnf(format string, args ...interface{}) {
 	HadWarnings = true
 }
 
-func loadCompose(filePath string, projectName string, overrideProjectName bool) (*compose.Project, error) {
-	// The default path for a Compose file is compose.yaml (preferred) or compose.yml that is placed in the working directory.
+func getComposeFilePath(userSpecifiedComposeFile string) (string, error) {
+	// The Compose file is compose.yaml (preferred) or compose.yml that is placed in the current directory or higher.
 	// Compose also supports docker-compose.yaml and docker-compose.yml for backwards compatibility.
-	if files, _ := filepath.Glob(filePath); len(files) > 1 {
-		return nil, fmt.Errorf("multiple Compose files found: %q; use -f to specify which one to use", files)
-	} else if len(files) == 1 {
-		filePath = files[0]
+	// Users can override the file by specifying file name
+	const DEFAULT_COMPOSE_FILE_PATTERN = "*compose.y*ml"
+
+	Debug("Loading compose file: ", userSpecifiedComposeFile)
+
+	path, err := os.Getwd()
+	searchPattern := DEFAULT_COMPOSE_FILE_PATTERN
+	if len(userSpecifiedComposeFile) > 0 {
+		path = ""
+		searchPattern = userSpecifiedComposeFile
+	}
+
+	if err != nil {
+		return path, fmt.Errorf("unable to get working directory for compose file")
```

Comment:
Here we're "hiding" the original error `err`; consider adding a `Debug` line or make it part of the new error by adding it in the formatting string.